builders/build/prerequisites.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Prerequisites
```

Comment:
I'd change this to "Prerequisites for building a ContainerChain" so it provides more context as a search result